forgerock-auth/src/main/java/org/forgerock/android/auth/RemoteWebAuthnRepository.kt

```diff
@@ -0,0 +1,137 @@
+/*
+ * Copyright (c) 2024 ForgeRock. All rights reserved.
+ *
+ *  This software may be modified and distributed under the terms
+ *  of the MIT license. See the LICENSE file for details.
+ */
+
+package org.forgerock.android.auth
+
+import android.net.Uri
+import kotlinx.coroutines.Dispatchers
+import kotlinx.coroutines.suspendCancellableCoroutine
+import kotlinx.coroutines.withContext
+import okhttp3.Call
+import okhttp3.Callback
+import okhttp3.Request
+import okhttp3.Response
+import org.forgerock.android.auth.exception.ApiException
+import org.forgerock.android.auth.webauthn.PublicKeyCredentialSource
+import org.forgerock.android.auth.webauthn.WebAuthnRepository
+import org.json.JSONObject
+import java.io.IOException
+import java.net.HttpURLConnection
+import java.net.URL
+import kotlin.coroutines.resume
+import kotlin.coroutines.resumeWithException
+
+private val TAG = RemoteWebAuthnRepository::class.java.simpleName
+
+internal class RemoteWebAuthnRepository(val serverConfig: ServerConfig = Config.getInstance().serverConfig) :
+    WebAuthnRepository {
+
+    override suspend fun delete(publicKeyCredentialSource: PublicKeyCredentialSource) {
+        val credentialId = String(publicKeyCredentialSource.id)
+        val userId = String(publicKeyCredentialSource.userHandle)
+
+        val findResponse = find(userId, credentialId)
+        val resourceId = findResponse.getString("uuid")
```

Comment:
getString will throw exception, may consider to use optString

Reply:
good catch